packages/client/src/components/designer/LayersMenu.tsx

```diff
@@ -125,8 +153,13 @@ export const LayersMenu = observer((): JSX.Element => {
 
     const [variableModal, setVariableModal] = useState('');
 
-    // get the active page
-    const activePage = state.blocks[designer.rendered];
+    const allBlockKeys = Object.keys(state.blocks);
```

Comment:
turn this into a filter, might be cleaner

Reply:
updated